tests/h/models/group_scope_test.py

```diff
@@ -47,9 +48,8 @@ def test_you_can_get_a_groupscopes_group_by_the_group_property(self, factories):
 
         assert group_scope.group == group
 
-    def test_you_can_get_a_groups_scopes_by_the_scopes_property(
-        self, factories, matchers
-    ):
+    @pytest.mark.usefixtures("matchers")
```

Comment:
```suggestion
```
I think the matchers are comparators. If we aren't using them it probably doesn't need to be here